simple/dgram.c

```diff
@@ -76,39 +76,23 @@ static int init_fabric(void)
 
 static int send_recv()
 {
-	struct fi_cq_entry comp;
 	int ret;
 
 	if (opts.dst_addr) {
 		fprintf(stdout, "Sending message...\n");
 		sprintf(buf, "Hello from Client!");
```

Comment:
With the move to common code, we should be able to add data verification to a lot more of the tests.  I'll make this change (good catch), but there really isn't a good reason for this test to validate data differently than other tests.  I'd rather move to using common data verification code.